spec/jobs/bulkrax/importer_job_spec.rb

```diff
@@ -30,7 +30,7 @@ module Bulkrax
         importer_job.perform(importer.id)
 
         expect(importer.current_run.total_work_entries).to eq(10)
-        expect(importer.current_run.total_collection_entries).to eq(427)
+        expect(importer.current_run.total_collection_entries).to eq(428)
```

Comment:
Curious how come this number ticked up by 1?

Reply:
Specs that reach out to external sources should always be stubbed. This one isn't, so the external count it's testing against changes sometimes. It needs to be properly stubbed, but no one has gotten around to doing it yet.